src/plone/api/content.py

```diff
@@ -187,7 +187,7 @@ def move(source=None, target=None, id=None, safe_id=False):
     # If no target is given the object is probably renamed
     if target and source.aq_parent is not target:
             target.manage_pasteObjects(
-                source.aq_parent.manage_cutObjects(source_id)
+                source.aq_parent.manage_cutObjects(source_id),
```

Comment:
@hvelarde no please!! 😄 Keep the parenthesis on its own, for two reasons:

- there is a clear clue where the statement finishes (as it will be aligned with the ``t`` of target
- if eventually someone needs to add a second parameter the diff will be just one single line, rather than two